app/src/main/java/fr/free/nrw/commons/achievements/Achievements.java

```diff
@@ -1,5 +1,7 @@
 package fr.free.nrw.commons.achievements;
 
+import android.util.Log;
+
 /**
```

Comment:
Javadocs are spotty for this class, please have a more descriptive Javadoc here, and add them to all the new methods as well. Thanks! :)

Reply:
@misaochan I have updated the javadocs :)